distgen/project.py

```diff
@@ -6,15 +6,24 @@ class AbstractProject():
     maintainer = "unknown <unknown@unknown.com>"
 
 
-    def inst_init(self, spec, template, config):
+    def inst_init(self, specfiles, template, sysconfig):
         """
-        Returns dict with data to be used in specification file.
+        Executed before the project.py/spec files/template is loaded and
+        before all the dynamic stuff and specification is calculated.
+        Now is still time to dynamically change the list of specfiles or
+        adjust the system configuration.  Any object variable (defined
```

Comment:
How does one define a variable?

Reply:
`self.x = "y"` means that `{{ project.x }}` will expand to `y`.  By object I meant the object instantiated from `Project`.